### DISCUSSION TITLE
Wallet connect not working

### DISCUSSION THREAD

**Discussion by shan-c** [WalletConnect/walletconnect-monorepo#2698] [Developer Support] [answered]
None of the wallets that implemented wallet connect v2 are working. It was working fine a few hours back.

**Comment by Loganistic** [comment]
Same thing on my side.

**Reply by m4kgsjd** [reply]
Same here too.

**Comment by Tomasvrba** [comment]
Same, getting a 404 on GET https://verify.walletconnect.com/attestation/...

**Comment by finessevanes** [comment, marked as answer]
@shan-c hey, we're aware of an issue that was impacting certain users in specific regions. This has been resolved.
https://status.walletconnect.com/
